test/multi_region_aws_cluster_scaling_test.go

```diff
@@ -0,0 +1,633 @@
+package test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"multiregiontests/internal/helpers"
+	kubectlHelpers "multiregiontests/internal/helpers/kubectl"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/stretchr/testify/require"
+)
+
+// TestAWSClusterScaling_ScaleBrokers tests scaling Zeebe brokers in a multi-region setup
+// Initial state: 8 brokers (4 per region), 8 partitions
+// Target state: 12 brokers (6 per region), 8 partitions
+// Reference: https://docs.camunda.io/docs/self-managed/components/orchestration-cluster/zeebe/operations/cluster-scaling/
+func TestAWSClusterScaling_ScaleBrokers(t *testing.T) {
+	t.Log("[CLUSTER SCALING TEST] Testing Zeebe broker scaling in multi-region mode 🚀")
+
+	if globalImageTag != "" {
+		t.Log("[GLOBAL IMAGE TAG] Overwriting image tag for all Camunda images with " + globalImageTag)
+		baseHelmVars = helpers.OverwriteImageTag(baseHelmVars, globalImageTag)
+	}
+
+	// Runs the tests sequentially
+	for _, testFuncs := range []struct {
+		name  string
+		tfunc func(*testing.T)
+	}{
+		{"TestInitKubernetesHelpers", initKubernetesHelpers},
+		{"TestVerifyInitialBrokerCount", verifyInitialBrokerCount},
```

Comment:
could in theory combine `verifyInitialBrokerCount` with `verifyScaledBrokerTopology` and pass in expected clusterSize and PartitionsCount as arguments.